backend/api/Models/Update/BaseModel.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Pims.Api.Models.Update
```

Comment:
Remind me what the benefit is of having all of the different 'flavours' of these model classes? We already have a BaseModel, this version just gets rid of a few fields. Could we just ignore those extra fields, or does that cause issues?

Reply:
There are a number of approaches to reducing duplication of models.  One of them is just to reuse as much as possible and let extra fields just get passed around.  This is a leaky model and results in an outwardly ugly api.  When we add Swagger our models would have properties that are irrelevant.

The downside the a cleaner outward api regrettably results in an uglier inward api.  Essentially creating models for every endpoint or groups of endpoints.  I lean towards this for the follow reasons (even though I hate the duplicate too)

- All endpoints (groups of endpoints) are self contained and won't contaminate or break each other when a model is changed
- API input and output is valid and relevant to the endpoints
- Models are lightweight and generally don't contain any logic, thus are easy enough to maintain.

The main downside is that when an entity is changed it **may** require updating multiple models to support the change.  While this is a risk, it is mitigated by the following;

- Unit tests should catch any breaking changes
- We don't always want to change the shape of all endpoint input/output when an entity changes